src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NET6_0
+global using global::System;
+global using global::System.Collections.Generic;
+global using global::System.Linq;
+global using global::System.Net.Http;
+global using global::System.Net.Http.Json;
+global using global::System.Threading.Tasks;
+global using global::Microsoft.AspNetCore.Builder;
+global using global::Microsoft.AspNetCore.Hosting;
+global using global::Microsoft.AspNetCore.Http;
+global using global::Microsoft.AspNetCore.Mvc;
+global using global::Microsoft.AspNetCore.Routing;
+global using global::Microsoft.Extensions.Configuration;
+global using global::Microsoft.Extensions.DependencyInjection;
+global using global::Microsoft.Extensions.Hosting;
+global using global::Microsoft.Extensions.Logging;
+#else
+#error Target frameworks need to be updated.
```

Comment:
@JunTaoLuo does this file show up in the solution explorer? Do we want it to?

cc @DamianEdwards

Reply:
That's a good point. I'll see what I can do here. Theoretically, we don't want the users to edit this at all but I don't know how we'd make a cs file uneditable. Alternatively, we can write a MSBuild task that generates this file and include it as a dll. That would make this file uneditable and nothing would show up in the solution explorer.

Reply:
Yes, look at what is being done today for assembly info, and do that. Try this experience though.

Reply:
Those are generated by code, I think. I saw the auto-generated headers but I can confirm tomorrow.

Reply:
File removed. It's now essentially the same logic as AssemblyInfo.cs. It doesn't show up in the solution explorer and is not editable.